### PR TITLE
MacAddress: Convert to template

### PR DESCRIPTION
**Pull request by mintsoft**
###### Description of changes

ref: https://github.com/duckduckgo/zeroclickinfo-goodies/issues/2767

---

Instant Answer Page: https://duck.co/ia/view/mac_address

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mogigoma

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by mintsoft** [comment]
I've changed the styling a bit here @mogigoma; hope it meets with your approval!

**Comment by GuiltyDolphin** [comment]
@moollaza It'd be good if we could get this on Beta.

**Comment by moollaza** [comment]
Deploying to beta...

File: lib/DDG/Goodie/MacAddress.pm [hunk 1]
```diff
@@ -39,20 +39,26 @@ handle remainder => sub {
...
+        data => {
+            name   => $owner,
+            result => \@lines,
+            input  => fmt_mac($_)
+        },
+        templates => {
+            options => {
+                content => 'DDH.mac_address.content',
+            },
+            group => 'text'
```
Comment (moollaza) [inline lib/DDG/Goodie/MacAddress.pm]: This looks a little odd, https://beta.duckduckgo.com/?q=mac+address+14%3AD6%3A4D%3ADA%3A79%3A6A&ia=answer

We should be using the `list` template here:

title: {{name}}
subtitle: "MAC Address Lookup: {{input}}
list_content: `<li>{{this}}</li>`

Comment (moollaza) [inline lib/DDG/Goodie/MacAddress.pm]: Hmm It looks pretty odd already -- I guess this is OK.

We should be using our `zci__` classes at least in the header + subtitle.

If these addresses are all equally detailed, we could try a map view as a V2

Comment (mintsoft) [inline lib/DDG/Goodie/MacAddress.pm]: Yeah I went for emulating the existing look n feel; just to clarify you mean the h3 h5 should have `zci__mac_address` on them?

Comment (moollaza) [inline lib/DDG/Goodie/MacAddress.pm]: Sorry meant the `c-base__title` and `c-base__subtitle` classes we use fur the generated Title and Subtitle in templates

Comment (mintsoft) [inline lib/DDG/Goodie/MacAddress.pm]: @moollaza Those classes make the text huge :/

**Comment by mintsoft** [comment]
@moollaza Took another stab, ignoring how it looks in prod right now; how about this instead:
macaddressmaybeimproved

**Comment by moollaza** [comment]
Deploying to Beta...

**Comment by GuiltyDolphin** [comment]
Ping @moollaza you happy with this?

**Comment by moollaza** [comment]
@mintsoft Thanks! This LGTM :+1:

**Pull request merged**